packages/InteractiveProfilingTool-Tests.package/DummyObject.class/instance/dummyMethodToRemove.st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
+dummyMethodToRemove
+   	^ 1
```

Comment:
Please consider adding a newline after the function name to be consistent with other methods.